services/app-api/src/resolvers/submitHealthPlanPackage.test.ts

```diff
@@ -256,6 +260,12 @@ describe('submitHealthPlanPackage', () => {
 
         const programs = [defaultFloridaProgram()]
         const name = packageName(sub, programs)
+        const stateAnalystsEmails = getTestStateAnalystsEmails(sub)
+
+        const cmsEmails = [
+            ...config.cmsReviewSharedEmails,
+            ...stateAnalystsEmails,
```

Comment:
can you also add a line here removing any non unique emails? That way we ensure we dont email people twice or attach their name twice.

Reply:
That sounds great. I'll merge this in this PR and tack on making this helper function to [MR-2361](https://qmacbis.atlassian.net/browse/MR-2361)